extension/browserUtil.js

```diff
@@ -8,6 +8,14 @@ export async function activeTab() {
   }))[0];
 }
 
+export async function activeTabs() {
```

Comment:
This name doesn't make sense for the function – the function returns all tabs except the active/pinned tabs. But I don't think we need a function for this anyway, it can just go directly in the intent.